src/AWS.Deploy.CLI/ServerMode/Controllers/DeploymentController.cs

```diff
@@ -36,6 +36,8 @@ namespace AWS.Deploy.CLI.ServerMode.Controllers
     [Route("api/v1/[controller]")]
     public class DeploymentController : ControllerBase
     {
+        private const string REPLACE_TOKEN_STACK_NAME = "{StackName}";
```

Comment:
The constant should not be defined more then once.

Reply:
I have only defined this once in this class.

Reply:
It is also defined in `DeployCommand` which is the same project.

Reply:
Fixed